firebase-perf/src/main/java/com/google/firebase/perf/injection/modules/GaugeModule.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.injection.modules;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import dagger.Module;
+import dagger.Provides;
+
+/**
+ * Provider for {@link com.google.firebase.perf.internal.GaugeManager} and related classes.
+ *
+ * @hide
```

Comment:
Why do we have this `@hide` annotation?

Reply:
It's historical. All of our javadocs are annotated with `@hide` and I don't why.

I have a b/177448055 to revisit our use of annotations.